specification/saas/resource-manager/Microsoft.SaaS/preview/2018-03-01-beta/saas.json

```diff
@@ -73,6 +73,45 @@
         }
       }
     },
+    "/subscriptions/{subscriptionId}/providers/Microsoft.SaaS/eligibilityCheck": {
```

Comment:
>eligibilityCheck [](http://example.com/codeflow?start=62&length=16)

Pl get an approval from the breaking changes reviewers for this change after which we can proceed with the ARM review

Reply:
Hi @Roopesh Manda - I added a new resource type "".,why it is count as breaking change?
According to [ARM Breaking changes document](https://microsoft.sharepoint.com/:w:/t/azureresourcemanagerteam/EWXsAQ1yx25KkyYCeeWGUgwBSBxEdUDEbHi6FZ__U8EOQw?rtime=VbIj7ihl1kg), this is not a breaking change.

Reply:
Based on our offline conversation please get this PR approved for breaking change (since there is an update to an existing api-version spec) and I'll go ahead and sign off your PR from the ARM side.